pillar/apps/odlvideo.sls

```diff
@@ -88,6 +88,8 @@ django:
     ET_PIPELINE_ID: {{ env_data.transcode_pipeline_id }}
     ET_PRESET_IDS: 1504127981769-6cnqhq,1504127981819-v44xlx,1504127981867-06dkm6,1504127981921-c2jlwt
     GA_DIMENSION_CAMERA: dimension1
+    GA_KEYFILE_JSON: {{ salt.vault.read('secret-odl-video/{env}/ga-keyfile-json'.format(env=ENVIRONMENT)).data.value }}
+    GA_VIEW_ID: {{ salt.vault.read('secret-odl-video/{env}/ga-view-id'.format(env=ENVIRONMENT)).data.value }}
```

Comment:
Is this actually sensitive information?

Reply:
GA_VIEW_ID is not sensitive. I moved it into the env_data, updated PR.